tests/fixtures/inference.py

```
@@ -72,3 +77,101 @@ def qwen_isvc(
+        ],
+        multi_model=False,
+        supported_model_formats=[{"autoSelect": True, "name": LLMdInferenceSimConfig.name}],
+    ) as serving_runtime:
+        yield serving_runtime
+
+
+@pytest.fixture(scope="class")
+def llm_d_inference_sim_isvc(
+    admin_client: DynamicClient,
+    model_namespace: Namespace,
+    minio_pod: Pod,
```

Ruff (0.14.0)

Unused function argument: `minio_pod`

(ARG001)